src/util/apis/generated/models/index.d.ts

```diff
@@ -8431,6 +8432,18 @@ export interface TestReportStats {
   devicesFailed: number;
   devicesSkipped: number;
   stepCount: number;
+  artifacts?: TestArtifacts;
+}
+
+/**
+ * @class
+ * Initializes a new instance of the TestArtifacts class.
+ * @constructor
+ * @member {string} nunit_xml_zip
```

Comment:
I think this is autogenerated? So I guess there's nothing we can do to make this more generic? I.e. so that when other frameworks add their equivalents to the `nunit_xml_zip` key we don't have to regenerate this Interface?

Reply:
> I think this is autogenerated?

No, I've added it manually

Reply:
I think this is meant to be autogenerated:

> Autorest-generated client code for talking to Bifrost.

Although I'm not sure what the process is for generating. Let me see if I can work it out - if not I'll see if @christav can point us in the right direction.

Reply:
OK - you should be able to run this command to generate the model: `npm run autorest` just testing it now.

Reply:
Looks like this removes the reference to the `artifacts` so I think we need to find a way to add this to the Bifrost API swagger so that the auto generation routine works correctly (otherwise another dev will remove this change next time the API code is regenerated).

Reply:
@erychagov - I'm looking in to this now. I've put in a PR to update the Swagger YAML file. I've made it slightly more generic than your example here so we may need to tweak the code once it's available (it'll be a Dictionary rather than referencing `nunit_xml_zip` by name).

Reply:
No manual changes to these files will be merged. These have to be autogenerated.

Reply:
For some reason the property we need wasn't in the public API swagger yaml. Just waiting for that to be sorted so we can autogenerate.